src/main/java/com/github/instagram4j/instagram4j/actions/users/UsersAction.java

```diff
@@ -25,4 +27,21 @@
         return new UsersInfoRequest(pk).execute(client)
                 .thenApply(UserResponse::getUser);
     }
+
+    public CompletableFuture<UsersSearchResponse> search(@NonNull String query) {
+        return new UsersSearchRequest(query).execute(client);
+    }
+
+    public CompletableFuture<UsersSearchResponse> search(@NonNull String query,@NonNull String search_surface, int timezone_offset, int count) {
+        return new UsersSearchRequest(query, search_surface, timezone_offset, count).execute(client);
+    }
+    
+    public CompletableFuture<UsersSearchResponse> getSearchNextPage(@NonNull UsersSearchResponse usersSearchResponse,@NonNull String query) {
```

Comment:
I don't believe it is necessary to add these getSearchNextPage() methods. 

The search() methods are perfectly fine, but I feel like this one method is really specific, and although it may be useful, I want to avoid the clutter.

If someone wants to get the next search results or get "x" page search they can do so using iterators.

Reply:
Alright I see!

For example, without my methods to get the page X we would need to do something like:
```Java
@SneakyThrows
private UsersSearchResponse getPage(String query, int page)
{
    UsersSearchResponse usersSearchResponse = igClient.actions().users().search(query).get();
    for(int i = 0 ; i < page; i++)
    {
        usersSearchResponse = new UsersSearchRequest(
            query,
            usersSearchResponse.getRank_token(),
            usersSearchResponse.getPage_token()
        ).execute(igClient).get();
    }
    return usersSearchResponse;
}
```